lms/static/scripts/frontend_apps/utils/api.js

```diff
@@ -90,6 +94,36 @@ async function apiCall({ path, authToken, data }) {
   return resultJson;
 }
 
-// Separate export from declaration to work around
-// https://github.com/robertknight/babel-plugin-mockable-imports/issues/9
-export { apiCall };
+/** @param {number} ms */
+function delay(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
```

Comment:
Could you add a comment to this function describing what it's for and maybe indicate that it's temporary?

Reply:
`delay` is a helper that we reproduce quite widely in tests to wait for a fixed amount of time. I'll add a comment to `fetchBooks` and `fetchChapters` to be more explicit that they are fakes/stubs.